### PR TITLE
#182991558 Created The Pricing Page

### PR DESCRIPTION
**Pull request by RACHID-WEBDEV**
#### What does this PR do?
Created The Pricing Page
#### Description of Task to be completed?
- Created the  `Pricing Component` 
- Added respective SVG icon to reach pricing 
- make typo changes in the  `Button component` 

#### How should this be manually tested?
- Clone the Repository
- Run `yarn Install` in your terminal to install required dependencies
-  Run `yarn dev` to start the local development server on `localhost:3000/pricing`
below is the image preview

pricing-2022-08-15-23_55_53

### CODE REVIEW

File: pages/pricing.js [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+
+            <div className=""></div>
+          </div>
+        </div>
+      </div>
+      <div className="bg-[#F5F6F8] px-14 py-16 lg:py-20 ">
+        <div>
+          <div className="mt-2  avatar-group text-center">
+            <div className="relative z-20 inline-flex items-center justify-center w-12 h-12 lg:w-14 lg:h-14 -ml-4  transition-all duration-200 border-2 border-white border-solid rounded-full ease-soft-in-out text-size-xs hover:z-40">
+              <img
```
Comment (andela-hpopoola) [inline pages/pricing.js:94]: Can we use the normal NextJS image here or change these to local images?

File: pages/pricing.js [hunk 2]
```diff
@@ -0,0 +1,135 @@
+/* eslint-disable @next/next/no-img-element */
```
Comment (andela-hpopoola) [inline pages/pricing.js:1]: Lets try to fix the image element eslint error and remove this

**Review by andela-hpopoola — APPROVED** [review]
Thank you for working on this. Here are the errors I noticed from the UI:

Screenshot 2022-08-16 at 6 29 29 AM

Feel free to merge after fixing the highlighted errors. Thank you.

File: pages/pricing.js (hunk 1 above)
Comment (RACHID-WEBDEV) [reply pages/pricing.js]: Fixed

**Pull request merged**